src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1349,6 +1349,39 @@
           text: az aks nodepool delete --resource-group MyResourceGroup --cluster-name MyManagedCluster --name nodepool1 --ignore-pod-disruption-budget=true
 """
 
+helps['aks nodepool operation-abort'] = """
+    type: command
+    short-summary: Abort last running operation on nodepool.
+    parameters:
+        - name: --resource-group
+          type: string
+          short-summary: Name of the resource group
+        - name: --cluster-name
+          type: string
+          short-summary: Name of the managed cluster
+        - name: --name -n
```

Comment:
```suggestion
        - name: --nodepool-name --name -n
```

This might still fail the CI, there's a strict rule on the order of these options and I'm not sure about the correct order.